graphql/message_translation/fields.ts

```diff
@@ -0,0 +1,22 @@
+import { MessageTemplateType, NotificationMessageType } from '../types/notificationMessage';
+import { Authorized, FieldResolver, Resolver, Root } from 'type-graphql';
+import { Message_translation as MessageTranslation } from '../generated';
+import { getNotification, getSampleContext } from '../../common/notification/notification';
+import { renderTemplate } from '../../utils/helpers';
+import { Role } from '../roles';
+
+@Resolver((of) => MessageTranslation)
+export class ExtendedFieldsMessageTranslationResolver {
+    @FieldResolver((returns) => NotificationMessageType)
+    @Authorized(Role.ADMIN)
+    async sampleMessage(@Root() messageTranslation: MessageTranslation): Promise<NotificationMessageType> {
```

Comment:
FYI @jeffersondarcy: I've added some utilities here to have better tooling for maintaining messages in the retool